ui/src/main/java/com/nextcloud/android/common/ui/theme/utils/AndroidViewThemeUtils.kt

```diff
@@ -80,7 +80,7 @@ class AndroidViewThemeUtils @Inject constructor(schemes: MaterialSchemes, privat
         }
     }
 
-    fun colorNavigationView(navigationView: NavigationView) {
+    fun colorNavigationView(navigationView: NavigationView, colorIcons: Boolean = true) {
```

Comment:
This is actually a breaking change for Java clients (as Java does not support optional arguments). You'll need to add the `@JvmOverloads` annotation to the method to prevent that, which will generate both signatures (with and without the optional argument).

Reply:
Resolved in https://github.com/nextcloud/android-common/pull/69/commits/299eb7dbb59d9d53100a8d0289a69ba5048a41ec

Reply:
Right, i missed that. Thanks!